tests/unit/charms/magma_orchestrator_interface/v0/test_magma_orchestrator_interface_requirer.py

```diff
@@ -56,44 +58,16 @@ def test_given_orchestrator_information_in_relation_data_when_relation_changed_t
         args, _ = patch_on_orchestrator_available.call_args
         orchestrator_available_event = args[0]
         self.assertEqual(orchestrator_available_event.root_ca_certificate, root_ca_certificate)
+        self.assertEqual(
+            orchestrator_available_event.certifier_pem_certificate, certifier_pem_certificate
+        )
         self.assertEqual(orchestrator_available_event.orchestrator_address, orchestrator_address)
         self.assertEqual(orchestrator_available_event.orchestrator_port, orchestrator_port)
         self.assertEqual(orchestrator_available_event.bootstrapper_address, bootstrapper_address)
         self.assertEqual(orchestrator_available_event.bootstrapper_port, bootstrapper_port)
         self.assertEqual(orchestrator_available_event.fluentd_address, fluentd_address)
         self.assertEqual(orchestrator_available_event.fluentd_port, fluentd_port)
 
-    @patch(f"{BASE_CHARM_DIR}._on_orchestrator_available")
-    def test_given_orchestrator_information_in_relation_data_when_relation_changed_and_schema_validation_fails_then_orchestrator_available_event_is_not_emitted(  # noqa: E501
```

Comment:
Why was this test removed?

Reply:
Because we removed the `uri` format from the JSON schema, so now it accepts any string. That's not great, but on the other hand, `uri` enforces wrong format. AGW expects endpoints in a form of a protocol relative URL, while `uri` requires protocol. There's no format suitable for this case.